### DISCUSSION TITLE
How to autenticate JWT?

### DISCUSSION THREAD

**Discussion by heiytor** [appwrite/appwrite#4861] [Q&A] [answered]
Hi, i'm trying to access `/v1/storage/buckets/{bucketId}/files/{fileId}/view` GET route from insomnia, but, i need to put a JWT token in headers. How can i generate a valid one?

**Comment by stnguyen90** [comment, marked as answer]
@heiytor you can generate a JWT token with the [Create JWT API](https://appwrite.io/docs/client/account?sdk=web-default#accountCreateJWT). You'll need to have a session before you can generate the JWT.
